src/greatwall/resources/knowledge/fractal.py

```diff
@@ -129,18 +131,25 @@ def burningship_set(
         x = np.linspace(x_min, x_max, width)
         y = np.linspace(y_min, y_max, height)
 
+        self.computed_values = {}
```

Comment:
To increase the hit rates I think you don't need to reset this attribute here. What do you think?

Reply:
I agree. I left the variable initialization in the constructor. I should check the PR first myself because this is something I should have seen, thanks.